gnc/navigator_perception/include/navigator_vision_lib/model.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MODEL_H
+#define MODEL_H
+#include <string>
+#include <vector>
+
+class Model
```

Comment:
This seems like its a model for the scan the code object specifically, you should specify it as such.